apps/src/publicKeyCryptography/NumberedSteps.jsx

```diff
@@ -17,15 +17,38 @@ export default function NumberedSteps(props) {
   return (
     <table>
       <tbody>
-        {React.Children.map(props.children, (child, index) => (
-          <tr>
-            <td style={style.td}>{(index + 1) + ')'}</td>
-            <td style={style.td}>{child}</td>
-          </tr>
-        ))}
+        {React.Children.map(props.children, (child, index) => {
+          const stepRequirements = child.props && child.props.requires || [];
```

Comment:
it seems a bit odd that this logic lives here and not in Step; any reason why `trStyle` cannot instead be applied to the `div` in `Step`'s render method?